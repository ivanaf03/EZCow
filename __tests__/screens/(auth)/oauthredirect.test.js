import React from "react";

import {render} from "@testing-library/react-native";

import OauthRedirect from "../../../app/(auth)/oauthredirect";

describe("OauthRedirect", () => {
    beforeEach(() => {
        jest.clearAllMocks();
    });

    it("should render correctly", () => {
        const tree = render(<OauthRedirect />);
        expect(tree).toMatchSnapshot();
    });

});