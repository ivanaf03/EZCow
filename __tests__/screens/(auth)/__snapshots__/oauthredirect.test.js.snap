// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OauthRedirect should render correctly 1`] = `
<View
  style={
    [
      {
        "flexBasis": "0%",
        "flexGrow": 1,
        "flexShrink": 1,
      },
      {
        "backgroundColor": "#1c1c1c",
      },
      {
        "justifyContent": "center",
      },
      {
        "alignItems": "center",
      },
    ]
  }
>
  <View>
    <View
      style={
        [
          {
            "display": "flex",
          },
          {
            "alignItems": "center",
          },
          {
            "justifyContent": "center",
          },
          {
            "paddingBottom": 16,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 16,
          },
          {
            "backgroundColor": "#272727",
          },
          {
            "borderLeftWidth": 2,
            "borderRightWidth": 2,
          },
          {
            "borderBottomColor": "#6bc0eb",
            "borderLeftColor": "#6bc0eb",
            "borderRightColor": "#6bc0eb",
            "borderTopColor": "#6bc0eb",
          },
          {
            "borderBottomLeftRadius": 8,
            "borderBottomRightRadius": 8,
            "borderTopLeftRadius": 8,
            "borderTopRightRadius": 8,
          },
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#ffffff",
            },
            {
              "fontSize": 24,
              "lineHeight": 32,
            },
            {
              "fontFamily": "Nunito-MediumItalic",
            },
          ]
        }
      >
        Iniciando sesi√≥n... Esto puede tardar unos segundos
      </Text>
    </View>
    <ActivityIndicator
      color="c_light_blue"
      size="large"
      style={
        [
          {
            "marginTop": 32,
          },
        ]
      }
    />
  </View>
</View>
`;
